readme.md

```diff
@@ -268,6 +268,16 @@ Returns a set of supported file extensions.
 
 Returns a set of supported MIME types.
 
+### FileType.addDetection(detection)
+
+Register a detection function. This function will be called after `file-type` and other detections complete.
```

Comment:
What other detections?

Reply:
Proprietary software file extensions that are not well known

Reply:
Or even plain text files that the users will be willing to support. Say, there's not enough confidence to be able to detect an svg or HTML to include it in the core, but the users could accept that and provide their own detection which will be enough for their use case.

Reply:
Yes, that's the whole point of this PR. That's not what I'm asking. I'm saying that "other detections" is not clear enough in the docs.

Reply:
I think "other" is probably intended to leave it open for extension.  Examples could be provided but are probably not going to be exhaustive.  Perhaps what you're asking for is some examples?